test/simulators_test.py

```diff
@@ -162,6 +193,25 @@ def test_send_measurement(self):
         self.cumulocity_api.delete_managed_object(device_id)
         log.info(f"Removed the {self.device_model.get('label')} with id {device_id}")
 
+    def test_shifplan(self):
+        log.info("Start testing create shiftplan")
+        # Create shiftplan
+        shiftplans = list(map(lambda shiftplan_model: Shiftplan(shiftplan_model), self.shiftplans))
+        for shiftplan in shiftplans:
+            try:
+                # Get created shiftplan info
+                shiftplan_info = self.oee_api.get_shiftplan(locationId=shiftplan.locationId, dateFrom=None, dateTo=None)
```

Comment:
- isnt there a way to just use `self.oee_api.get_shiftplan(locationId=shiftplan.locationId)` ?
- I think that should be possible if we use default parameters in the method definition: https://www.geeksforgeeks.org/default-arguments-in-python/
- the default values for dateFrom & dateTo should be None